src/main/java/core/basesyntax/model/Operation.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.model;
+
+public enum Operation {
+    BALANCE("b"), SUPPLY("s"), PURCHASE("p"), RETURN("r");
```

Comment:
```suggestion
    BALANCE("b"),
    SUPPLY("s"),
    PURCHASE("p"),
    RETURN("r");
```
Лучше сделать в столбик, для лучшей читаемости.

Reply:
i think it's ok

Reply:
let's make it a column, if our ENUM has constructor.